dev/src/main/java/org/eclipse/codewind/intellij/ui/module/CodewindModuleBuilder.java

```diff
@@ -139,8 +140,16 @@ public void moduleCreated(@NotNull Module module) {
         String language = template.getLanguage();
         String projectType = template.getProjectType();
         String conid = LocalConnection.CONNECTION_ID;
-
         Project ideaProject = module.getProject();
+
+        // The 'appsody init' command will fail to initialize the maven cache if there is no JDK on the PATH and
+        // JAVA_HOME is not set.  We set JAVA_HOME for the process running 'cwctl create' to ensure this doesn't happen.
+        Sdk sdk = ProjectRootManager.getInstance(ideaProject).getProjectSdk();
+        String javaHome = sdk == null ? null : sdk.getHomePath();
+        if (javaHome == null) {
+            Logger.log("createProject: no sdk set for project: " + ideaProject.getName());
```

Comment:
Hey John, if there is no SDK set then will we let the user proceed with createProject or should we force the user to set it before continuing?